src/style.css

```diff
@@ -1,3 +1,71 @@
+html {
+  font-family: Roboto, sans-serif;
+  min-width: 1023px;
+  max-width: 1201px;
+  height: 60px;
+}
+
+:root {
+  --color: #00acdc;
```

Comment:
The CSS variable `--color` is defined but not used in the styles. Consider using `var(--color)` for color properties where the blue color `#00acdc` is applied, such as in `.is-active` and `.nav__item:hover`.